src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -141,7 +144,7 @@ class CloseAccountPage extends Component {
                         </Text>
                     )}
                     onConfirm={CloseAccount.clearError}
-                    isVisible={Boolean(this.props.closeAccount.error)}
+                    isVisible={Boolean(this.props.closeAccount.error) || Boolean(!_isEmpty(this.props.closeAccount.errors))}
```

Comment:
You are missing the period for `_.isEmpty`. Also you don't need a boolean constructor because` _.isEmpty` returns a boolean.